pyproject.toml

```diff
@@ -47,32 +47,39 @@ dependencies = [
 
 [project.optional-dependencies]
 dev = [
-    "autoflake",
-    "black",
-    "deepdiff",
-    "flake8",
-    "isort",
-    "jupyter",
+    # Documentation
     "mkdocs",
     "mkdocs-material",
     "markdown-include",
     "mkdocs-redirects",
     "mkdocstrings[python] >= 0.18",
+    "jupyter",
```

Comment:
Did not move jupyter to match alphabetical order as it is "grouped" with `nbconvert` but I can move it too if you prefer

Reply:
makes sense, let's leave it like this 👍